### DISCUSSION TITLE
concepts/architecture/

### DISCUSSION THREAD

**Discussion by giscus** [go-gost/gost#486] [General]
# concepts/architecture/

https://latest.gost.run/concepts/architecture/

**Comment by kangiu** [comment]
这里要有张图就好了

**Comment by beast-whisperer** [comment]
太强了！

**Comment by tongwoo** [comment]
好拗口啊 >_<

**Comment by beck-8** [comment]
这个项目很喜欢，但概念真的多

**Comment by geekgao** [comment]
C4 架构图：
![Example Image](https://i.imgur.com/8ZQwSGA.png)
![Example Image](https://i.imgur.com/Up0VMrT.png)
